src/backends/caffe/caffemodel.cc

```diff
@@ -200,6 +200,8 @@ namespace dd
 	      {
 		std::vector<std::string> selts = dd_utils::split((*hit),'/');
 		fileops::copy_file((*hit),target_repo + '/' + selts.back());
+        if (selts.back().find(".json") != std::string::npos)
+          fileops::replace_string_in_file(target_repo + '/' + selts.back(), "db:true", "db:false");
```

Comment:
The `db` from json in file should have quotes. Have you been able to test the PR by publishing a model by any chance ?

Reply:
corrected, sorry for that